### PR TITLE
refactored test_admin_integration.py

### PR DESCRIPTION
**Pull request by PrashansaChaudhary**
**Problem**

What problem have you solved?
Refactored the test_admin_integration.py

**Solution**

How did you solve the problem?
Segregated tests based on their functionality and logical grouping

**Acceptance Criteria**

Have you written tests? Have you included screenshots of your changes if applicable?
Did you document your changes?

### CODE REVIEW

**Comment by matthewhegarty** [comment]
Hi - thanks for resubmitting.  I've converted the PR to 'draft' whilst it is still in progress.

**Comment by matthewhegarty** [comment]
Please can you install [pre-commit](https://github.com/django-import-export/django-import-export/blob/main/docs/contributing.rst#development) and then code will be formatted correctly when you commit.

File: tests/core/tests/admin_integration/test_action.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+from django.test.utils import override_settings
+
+from import_export.admin import (
+    ExportActionModelAdmin,
+    ExportMixin,
+    ImportExportActionModelAdmin,
+)
+from import_export.formats import base_formats
+from import_export.tmp_storages import TempFolderStorage
+from tests.core.tests.admin_integration.test_admin_integration import AdminTestMixin
```
Comment (matthewhegarty) [inline tests/core/tests/admin_integration/test_action.py:18]: I think it is better to import as follows:

```
from core.tests.admin_integration.test_admin_integration import AdminTestMixin
```

(applies to other files as well)

File: tests/core/tests/admin_integration/test_admin_integration.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import os
+
+from core.admin import BookAdmin
+from django.contrib.auth.models import User
+
+from import_export.formats.base_formats import DEFAULT_FORMATS
+
+
+class AdminTestMixin(object):
```
Comment (matthewhegarty) [inline tests/core/tests/admin_integration/test_admin_integration.py:9]: This file contains a mixin and no tests, therefore I think the file should be renamed as `mixins.py`

**Review by matthewhegarty — CHANGES_REQUESTED** [review]
Looks really good.  I have made a couple of minor comments.

**Comment by matthewhegarty** [comment]
@PrashansaChaudhary You can also update AUTHORS with your name if you like, and also updated changelog.rst

**Comment by matthewhegarty** [comment]
Looks great, nearly there... All that's left to do:

- Reformat the code with [`pre-commit`](https://github.com/django-import-export/django-import-export/pull/1662#issuecomment-1782520455), then push again
- Add your name or username to `AUTHORS` (optional)
- Add a row to `CHANGELOG.rst`, something like
  - 'Refactored `test_admin_integration`: split into smaller test modules (#1662)'

Let me know if you need any guidance with this.

File: AUTHORS [hunk 3]
```diff
@@ -2,146 +2,147 @@ The django-import-export was original created by Bojan Mihelac (bmihelac).
...
+-   bdnettleton (Brian Nettleton)
+-   Ptosiek (Antonin)
+-   samupl (Jakub Szafrański)
+-   smunoz-ml (Santiago Muñoz)
+-   carlosal0ns0 (Carlos Alonso)
+-   travenin (Lauri Virtanen)
+-   christophehenry (Christophe Henry)
+-   bgelov (Oleg Belov)
+-   EricOuma (Eric Ouma)
+-   Glay00 (Gleb Gorelov)
```
Comment (matthewhegarty) [inline AUTHORS:147]: It looks like you have modified all lines here, you should only add your name on a new line.  Please can you revert and submit again.

File: docs/changelog.rst [hunk 4]
```diff
@@ -17,6 +17,7 @@ Please refer to :doc:`release notes<release_notes>`.
 - Clarified ``skip_diff`` documentation (#1655)
 - Standardised interface of :meth:`~import_export.widgets.Widget.render` (#1657)
 - Improved documentation relating to validation on import (#1665)
+- Refactored test_admin_integration: split into smaller test modules (# 1662)
```
Comment (matthewhegarty) [inline docs/changelog.rst:20]: please remove the whitespace so that the line renders correctly i.e. change (# 1662) to (#1662)

**Review by matthewhegarty — CHANGES_REQUESTED** [review]
Some minor changes please

**Comment by PrashansaChaudhary** [comment]
I didnt mean to change the format for each line. My Vscode automatically formated the file which indirectly made the changes. I have made the requested changes to the file. Sorry for the trouble and thank you for your kind response

**Comment by matthewhegarty** [comment]
It's no trouble, thanks for taking the time to work on this.  All looks good.  Once the tests pass I will merge.

**Pull request merged**